app/javascript/components/Discover/Search.tsx

```diff
@@ -99,7 +99,7 @@ export const Search = ({ query, setQuery }: { query?: string | undefined; setQue
           {typeof item === "string" ? (
             <div {...props}>
               <a href={Routes.discover_path({ query: item })} className="flex flex-1 items-center no-underline">
-                <Icon name="clock-history" className="text-muted mr-2" />
+                <Icon name="clock-history" className="mr-2 text-muted" />
```

Comment:
A bunch of these were rearranged by Prettier since it's now a Tailwind class rather than a design system one